packages/svelte-ux/src/lib/components/ThemeSwitch.svelte

```diff
@@ -16,22 +25,34 @@
     let newTheme = e.target?.checked ? 'dark' : 'light';
     currentTheme.setTheme(newTheme);
   }}
-  classes={{
-    switch: 'dark:bg-primary dark:border-primary',
-    toggle: 'translate-x-0 dark:translate-x-full',
-  }}
+  classes={clsMerge(
+    {
+      switch: 'dark:bg-primary dark:border-primary',
+      toggle: 'translate-x-0 dark:translate-x-full',
+    },
+    settingsClasses.root,
+    classes
```

Comment:
it isn't explicit here but if someone did `<ThemeSwitch class="..." />` then the `class` would still get passed down to the underlying `Switch` via `$$restProps`

Reply:
that's correct